src/utils/sendWork.js

```diff
@@ -22,6 +23,9 @@ const sendWork = async (experimentId, timeout, body, requestProps = {}) => {
     uuid: requestUuid,
     socketId: io.id,
     experimentId,
+    extraHeaders: {
+      Authorization: 'Bearer admin',
```

Comment:
See previous comment about getting the header by importing it from another file.